src/models/Import.php

```diff
@@ -87,6 +87,7 @@ public function rules()
             ],
             [['file'], 'safe', 'on' => self::SCENARIO_INSERT],
             [['data'], 'validateCSV', 'on' => self::SCENARIO_INSERT],
+            [['user_id'], 'integer'],
```

Comment:
does this field already exist in the db?

Reply:
Not yet, so, I created a migrate at samurai project to add user_id to import table.

Reply:
@NgocTuanTruong can you move that migration from `samurai` to this package's migration file.